include/definitions.h

```diff
@@ -94,4 +105,18 @@
  * https://randomnerdtutorials.com/esp32-pinout-reference-gpios/
  */
 
+#define DEBUG true
+#define debug_printf(fmt, ...)                                                                                                             \
+  do {                                                                                                                                     \
+    if (DEBUG)                                                                                                                             \
+      fprintf(stderr, "%-32s:%3d %-36s: " fmt, __FILE__, __LINE__, __func__, __VA_ARGS__);                                                 \
+  } while (0)
+
+#define DEBUG2 false
+#define debug_printf_l2(fmt, ...)                                                                                                          \
+  do {                                                                                                                                     \
+    if (DEBUG2)                                                                                                                            \
+      fprintf(stderr, "%-32s:%3d %-36s: " fmt, __FILE__, __LINE__, __func__, __VA_ARGS__);                                                 \
+  } while (0)
```

Comment:
yes, good idea.
It would be great to make the Debug-Level a real variable (used in preprocessor but changeable in runtime)
--> in a new branch from master after merge this PR, o.k.?